README.md

```diff
@@ -270,6 +270,8 @@ You are ready to deploy:
 $ git push heroku master
 ```
 
+EmberCLI compilation happens at deploy-time, triggered by the `asset:precompilation` rake task.
```

Comment:
the `EmberCLI` can stay for now, but the `precompilation` is incorrect and should be fixed elsewhere. Thanks!

> It seems like a really nice architectural feature / benefit and worth telling people about.
I agree! I'm open to suggestions on where to mention it.